website/orders/templates/orders/order.html

```diff
@@ -8,12 +8,24 @@
 
 {% block page %}
     <div class="container-sm mb-5 text-center">
-        <h1>{{ shift.venue.name }} shift #{{ shift.pk }}</h1>
+        <h1>{{ shift.venue.name }}</h1>
+        <h3>
+            shift #{{ shift.pk }}
+            {% if shift.assignees %}
+                -
+                {% for assignee in shift.get_assignees %}
```

Comment:
Maybe a template tag for this whole description would be nice

Reply:
Created a template to extend.